### DISCUSSION TITLE
Use webpack settings from next.config.js

### DISCUSSION THREAD

**Discussion by StephanAksenchenko** [RyanClementsHax/storybook-addon-next#73] [Ideas]
- Hi! I'd like to suggest such feature as use webpack settings from next.config.js to storybook webpack config to exclude repeating.[here](https://github.com/RyanClementsHax/storybook-addon-next/blob/a7efc092c5db5b6e96c9dfeb478bca01567fe1c6/examples/svgr/.storybook/main.js#L19)

**Comment by RyanClementsHax** [comment]
Thanks for suggesting this!

I have considered doing this before, but I'm not sure this would be the best long term solution for most people. My reasoning is spelled out in great detail [here](https://github.com/RyanClementsHax/storybook-addon-next/issues/57#issuecomment-1086734218), but the TLDR is that the webpack modifications in `next.config.js` for any non-toy project likely won't work well when applied to storybook's config.

If you have any thoughts or comments I'd be happy to hear them! :)
